packages/gas-tank/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@latticexyz/gas-tank",
+  "version": "2.0.5",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/latticexyz/mud.git",
+    "directory": "packages/gas-tank"
+  },
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    "./mud.config": "./dist/mud.config.js",
+    "./worlds.json": "./worlds.json",
+    "./out/*": "./out/*"
+  },
+  "typesVersions": {
+    "*": {
+      "worlds.json": [
+        "./worlds.json"
+      ],
+      "mud.config": [
+        "./mud.config.ts"
+      ]
+    }
+  },
+  "scripts": {
+    "build": "pnpm build:js && pnpm build:mud",
+    "build:js": "tsup",
+    "build:mud": "mud build",
+    "clean": "pnpm run clean:abi && pnpm run clean:js && pnpm run clean:mud",
+    "clean:abi": "forge clean",
+    "clean:js": "rimraf dist",
+    "clean:mud": "rimraf src/codegen",
+    "deploy:garnet": "pnpm run build && mud deploy --profile=garnet",
+    "deploy:local": "pnpm run build && mud deploy --salt 0x",
+    "deploy:testnet": "pnpm run build && mud deploy --profile=lattice-testnet",
+    "dev": "mud dev-contracts --rpc http://127.0.0.1:8545",
+    "lint": "pnpm run prettier && pnpm run solhint",
+    "prettier": "prettier --write 'src/**/*.sol'",
+    "solhint": "solhint --config ./.solhint.json 'src/**/*.sol' --fix",
+    "test": "tsc --noEmit && mud test"
+  },
+  "dependencies": {
+    "@latticexyz/cli": "workspace:*",
+    "@latticexyz/schema-type": "workspace:*",
+    "@latticexyz/store": "workspace:*",
+    "@latticexyz/world": "workspace:*",
+    "@latticexyz/world-modules": "workspace:*"
+  },
+  "devDependencies": {
+    "@openzeppelin/contracts": "^5.0.0",
+    "@types/node": "^18.15.11",
+    "account-abstraction": "https://github.com/eth-infinitism/account-abstraction.git#v0.7.0",
```

Comment:
why do we use this git reference here instead of installing this via `"@account-abstraction": "0.7.0"` like we do in account-kit?

Reply:
an oversight! fixed